lib/board.rb

```diff
@@ -0,0 +1,61 @@
+class Board
+
+  attr_accessor :cells
+  #data and logic of the tictactoe board, stores state of board
+  
+  def initialize
+    @cells = reset!
+  end
```

Comment:
We could simplify to just `reset!`, as the assignment is made within that method.